docs/libs/examples/libs-example-custom-post-type.md

```diff
@@ -0,0 +1,157 @@
+---
+id: libs-example-custom-post-type
+title: Register Custom Post Type
+---
+
+While you are working on your WordPress project it is possible that at one point you will need a Custom Post Type to enrich your project.
+
+For that case we have prepared an easy way for you to register a new post type following these steps and with the following code example:
+
+First we will setup our folder structure:
+
+### Folder structure
+
+```php
+
+* src
+  * class-main.php
+  * custom-post-type
+    * class-example.php
+
+```
+PRO HINT: Files containing a class should be named class-{classname}.php and there should always be only one class per file.
```

Comment:
```suggestion
Files containing a class should be named `class-{classname}.php` and there should always be only one class per file.
```

I'd avoid a PRO word here. I think the PSR are using one class per file suggestion, and `class-{classname}.php` is a WordPress Coding Standard suggestion.